docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -227,4 +227,4 @@ Then the monitor will resume running according to the defined schedule.
 Using `retestOnFailure` can reduce noise related to transient problems.
 // end::monitor-config-options[]
 
-For information on configuring monitors individually, refer to <<synthetics-monitor-use>>.
+For information on configuring monitors individually, for browser monitors refer to <<synthetics-monitor-use>> and for lightweight monitors refer to <<synthetics-lightweight-data-types>>.
```

Comment:
There's a pile of up "for" clauses here that makes the sentence a little harder to scan...how about something like:
```suggestion
For information on configuring monitors individually, refer to:

* <<synthetics-monitor-use>> for browser monitors 
* <<synthetics-lightweight-data-types>> for lightweight monitors
```